javascript/operators/object_initializer.json

```diff
@@ -0,0 +1,277 @@
+            },
+            "chrome": {
+              "version_added": "1"
+            },
+            "chrome_android": {
+              "version_added": "1"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "1"
+            },
+            "firefox_android": {
+              "version_added": "1"
+            },
+            "ie": {
+              "version_added": "1"
+            },
+            "ie_mobile": {
+              "version_added": "1"
+            },
+            "nodejs": {
+              "version_added": true
+            },
+            "opera": {
+              "version_added": "1"
+            },
+            "opera_android": {
+              "version_added": "1"
+            },
+            "safari": {
+              "version_added": "1"
+            },
+            "safari_ios": {
+              "version_added": "1"
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "computed_property_names": {
+          "__compat": {
+            "description": "Computed property names",
+            "support": {
+              "webview_android": {
+                "version_added": true
+              },
+              "chrome": {
+                "version_added": true
+              },
+              "chrome_android": {
+                "version_added": true
+              },
+              "edge": {
+                "version_added": true
+              },
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": {
+                "version_added": "34"
+              },
+              "firefox_android": {
+                "version_added": "34"
+              },
+              "ie": {
+                "version_added": false
+              },
+              "ie_mobile": {
+                "version_added": false
+              },
+              "nodejs": {
+                "version_added": true
+              },
+              "opera": {
+                "version_added": "34"
+              },
+              "opera_android": {
+                "version_added": "34"
+              },
+              "safari": {
+                "version_added": "7.1"
+              },
+              "safari_ios": {
+                "version_added": "7.1"
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "shorthand_property_names": {
+          "__compat": {
+            "description": "Shorthand property names",
+            "support": {
+              "webview_android": {
+                "version_added": true
```

Comment:
If added in Opera 34, it should be 47 (as O34 is based on chrome 47): see http://www.opera.com/docs/history/